src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/ConvertToJsonCommand.cs

```diff
@@ -464,6 +478,12 @@ private object ProcessValue(object obj, int depth)
                     {
                         rv = obj.ToString();
                     }
+#if !CORECLR
```

Comment:
CORECLR uses a different mechanism to convert enums to strings (Newtonsoft.JSON has native support for this). If you look elsewhere in the PR you will see the enum converter conditionally applied for CORECLR. If you want this in comments in the code, I'll add them when I get back from vacation.